src/briefcase/platforms/macOS/__init__.py

```diff
@@ -37,12 +40,96 @@ class macOSMixin:
     supported_host_os_reason = "macOS applications can only be built on macOS."
 
 
+class macOSInstallMixin(AppPackagesMergeMixin):
+    def _install_app_requirements(
+        self,
+        app: AppConfig,
+        requires: list[str],
+        app_packages_path: Path,
+    ):
+        # Perform the initial install targeting the current platform
+        host_app_packages_path = (
+            self.bundle_path(app) / f"app_packages.{self.tools.host_arch}"
+        )
+        super()._install_app_requirements(
+            app,
+            requires=requires,
+            app_packages_path=host_app_packages_path,
+        )
+
+        # Find all the packages with binary components.
+        # We can ignore any -universal2 packages; they're already fat.
+        binary_packages = self.find_binary_packages(
+            host_app_packages_path,
+            universal_suffix="_universal2",
+        )
+
+        # Now install dependencies for the architecture that isn't the host architecture.
+        other_arch = {
+            "arm64": "x86_64",
+            "x86_64": "arm64",
+        }[self.tools.host_arch]
+
+        # Create a temporary folder to hold wheels for other targets
+        other_app_packages_path = self.bundle_path(app) / f"app_packages.{other_arch}"
+        if other_app_packages_path.is_dir():
+            self.tools.shutil.rmtree(other_app_packages_path)
+        self.tools.os.mkdir(other_app_packages_path)
+
+        if binary_packages:
+            with self.input.wait_bar(
+                f"Installing binary app requirements for {other_arch}..."
+            ):
+                try:
+                    self.tools[app].app_context.run(
+                        [
+                            sys.executable,
+                            "-u",
+                            "-X",
+                            "utf8",
+                            "-m",
+                            "pip",
```

Comment:
The common code between this call of pip, and the one in commands/create.py, should be factored out.